src/parser/WASMParser.cpp

```diff
@@ -2126,6 +2189,9 @@ class WASMBinaryReader : public wabt::WASMBinaryReaderDelegate {
 
     virtual void OnEndExpr() override
     {
+        // unify eqz in a block followed by a JumpIf
+        // can lead to wrong bytecode
+        m_lastEqzPos = SIZE_MAX - sizeof(Walrus::I32Eqz);
```

Comment:
I would introduce a named constant for `SIZE_MAX - sizeof(Walrus::I32Eqz)` and use it everywhere.

Reply:
Okay.